couch2pg/src/importer.js

```diff
@@ -5,7 +5,7 @@ import * as db from './db.js';
 const SELECT_SEQ_STMT = `SELECT seq FROM ${db.postgresProgressTable} WHERE source = $1`;
 const INSERT_SEQ_STMT = `INSERT INTO ${db.postgresProgressTable}(seq, source) VALUES ($1, $2)`;
 const UPDATE_SEQ_STMT = `UPDATE ${db.postgresProgressTable} SET seq = $1 WHERE source = $2`;
-const INSERT_DOCS_STMT = `INSERT INTO ${db.postgresTable} ("@timestamp", _id, _rev, doc) VALUES`;
+const INSERT_DOCS_STMT = `INSERT INTO ${db.postgresTable} (savedTimestamp, _id, _rev, doc) VALUES`;
```

Comment:
Can you use `saved_timestamp` as the column name here and throughout the code? as suggested in [cht-pipeline #99](https://github.com/medic/cht-pipeline/pull/99)